### PR TITLE
[Aggregator] correct precompute step

### PR DESCRIPTION
**Pull request by amyskov**
PR adds Pandas module to the `eval` function scope. With this change it will be possible to use complex lambdas with Pandas functions on the `precompute` step.

### CODE REVIEW

File: bearysta/aggregate.py [hunk 1]
```diff
@@ -454,11 +454,11 @@ def percent_diff_inv(column, **kwargs):
                     if 'row[' in func:
                         # Execute this as a row function
                         # Evaluate the lambda function on the df, passing in our locals
-                        result = df.apply(eval('lambda row:'+func, locals()), axis=1)
+                        result = df.apply(eval('lambda row:'+func, {"pd": pd}, locals()), axis=1)
```
Comment (anton-malakhov) [inline bearysta/aggregate.py:457]: isn't locals() become "globals" for the lambda environment?

Comment (amyskov) [reply bearysta/aggregate.py]: Wright, correctin was done. Also `read_csv_cached` function is added (regular `pandas.read_csv` with data caching feature). New function can be used on the precompute step, to do this just add `read_csv` inside your lambda with `pandas.read_csv` parameters. `precompute` section of config file can be set as it shown below:
```
# Replacement of rows in the column col_name in accordance of data in explain-col_name.csv file
precomputed:
    col_name: "read_csv('explain-col_name.csv', header=None, index_col=0, squeeze=True).to_dict().get(row['col_name'], row['col_name'])"
```

**Review by anton-malakhov — APPROVED** [review]
LGTM, thanks! Let's see what @bibikar says

**Comment by anton-malakhov** [comment]
I'm merging this but feedback is still welcome!

**Pull request merged**